templates/dashboard/company/list.html

```diff
@@ -0,0 +1,89 @@
+{% extends "dashboard/base.html" %}
+{% load i18n %}
+{% load materializecss %}
+{% load staticfiles %}
+{% load utils %}
+
+{% block title %}
+  {% trans "Companies" context "Dashboard companies list" %} - {{ block.super }}
+{% endblock %}
+
+{% block body_class %}body-companies{% endblock %}
+
+{% block menu_companies_class %}active{% endblock %}
+
+{% block header_extra %}
+  {% if perms.account.edit_staff %}
+    <div class="btn-fab-fixed">
+      <a href="{% url 'dashboard:company-create' %}" class="btn-fab-default waves-effect waves-light" title="{% trans "Add company" context "Company list add button text" %}" >
+        <svg data-src="{% static "dashboard/images/add.svg" %}" />
+      </a>
+    </div>
+  {% endif %}
+{% endblock %}
+
+{% block breadcrumbs %}
+  <ul class="breadcrumbs">
+    <li class="visible-s">
+      {% trans "Companies" context "Dashboard companies list" %}
+    </li>
+  </ul>
+{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col s12 l9">
+      {% if companies %}
+        <div class="card">
+          <div class="data-table-container">
+            <table class="bordered highlight responsive data-table">
+              <thead>
+                <tr>
+                  {% trans "Name" context "Company table header" as label %}
+                  {% sorting_header 'name' label %}
+
+                  {% trans "Location" context "Customers table header" as label %}
```

Comment:
I'm with ya. I'll remove it.

Reply:
Actually, I'd prefer to keep the filter. It does seem useful to me to easily generate a list of every customer that has a presence in a particular place. I'd probably change the filter so that it filters on all addresses associated with the company rather than just billing address. It also doesn't seem like a big deal to me so, if you're intent on getting rid of it, I'll drop it.